tests/Feature/Sensors/MailSensorTest.php

```diff
@@ -136,13 +136,7 @@ public function via(object $notifiable)
 
             public function toMail(object $notifiable): MailMessage
             {
-                app()->setBasePath(dirname(app()->basePath()).'/');
-                app()->useAppPath(app()->basePath('/src'));
-
-                return (new MailMessage)
-                    ->line('The introduction to the notification.')
-                    ->action('Notification Action', url('/'))
-                    ->line('Thank you for using our application!');
```

Comment:
Removing the need to do this setBasePath and useAppPath stuff here. I ran into this issue elsewhere and worked out how to fix it so I rolled the fix out here as well.